rocks-bin/src/purge.rs

```diff
@@ -0,0 +1,13 @@
+use eyre::Result;
+use rocks_lib::{config::Config, tree::Tree};
+
+pub fn purge(config: Config) -> Result<()> {
```

Comment:
question: would it make sense to prompt for confirmation by default, and add a `--skip-prompt`  flag?